### DISCUSSION TITLE
link with LAN(ip) in webclient

### DISCUSSION THREAD

**Discussion by lgzcoollg** [rustdesk/rustdesk#7917] [General]
link with LAN(ip) in webclient

**Comment by gamal3330** [comment]
![1](https://github.com/rustdesk/rustdesk/assets/60444970/25f1f290-6677-457a-bcee-57ce798d91f7)

**Reply by lgzcoollg** [reply]
the windows/Mac client could be . but what I mean is in the webclient.
